src/aposteriori/data_prep/cli.py

```diff
@@ -267,20 +268,20 @@ def cli(
         codec = Codec.CNO()
     elif atom_encoder == "CNOCB":
         codec = Codec.CNOCB()
-    elif atom_encoder == "CNOCBCA":
-        codec = Codec.CNOCBCA()
-    elif atom_encoder == "CNOCBCAQ":
-        codec = Codec.CNOCBCAQ()
-    elif atom_encoder == "CNOCBCAP":
-        codec = Codec.CNOCBCAP()
+    elif atom_encoder == "CNOCACB":
+        codec = Codec.CNOCACB()
+    elif atom_encoder == "CNOCACBQ":
+        codec = Codec.CNOCACBQ()
+    elif atom_encoder == "CNOCACBP":
+        codec = Codec.CNOCACBP()
     else:
         assert atom_encoder in [
             "CNO",
             "CNOCB",
-            "CNOCBCA",
-            "CNOCBCAQ",
-            "CNOCBCAP",
-        ], f"Expected encoder to be CNO, CNOCB, CNOCBCA, CNOCBCAQ, CNOCBCAP, but got {atom_encoder}"
+            "CNOCACB",
+            "CNOCACBQ",
+            "CNOCACBP",
+        ], f"Expected encoder to be CNO, CNOCB, CNOCACB, CNOCACBQ, CNOCACBP, but got {atom_encoder}"
```

Comment:
again here. 

I would say let's raise a warning if you are using the old encoders saying they are deprecated and will be removed in future versions.